lib/compress/fse_compress.c

```diff
@@ -205,26 +206,16 @@ size_t FSE_buildCTable_wksp(FSE_CTable* ct,
                 symbol, normalizedCounter[symbol],
                 FSE_getMaxNbBits(symbolTT, symbol),
                 (double)FSE_bitCost(symbolTT, tableLog, symbol, 8) / 256);
-        }
-    }
+    }   }
 #endif
 
     return 0;
 }
 
-#ifndef ZSTD_NO_UNUSED_FUNCTIONS
-size_t FSE_buildCTable(FSE_CTable* ct, const short* normalizedCounter, unsigned maxSymbolValue, unsigned tableLog)
```

Comment:
Is FSE_buildCTable expected to be deleted from .c file but not from the header file?

Reply:
No, it should be deleted from `*.h` file too.

What is happening is that we stop trying to keep parity between the stand-alone `libfse` version and the one integrated into `zstd`. All entry points that are not employed in `zstd` will be deleted within the near future, making this variant effectively no longer extractable.

What prompted the deletion of this specific entry point is that it was wrong and thus misleading : the amount of stack memory it allocates is no longer correct, as the underlying invoked function has changed. Using this function "as is" would lead to stack memory corruption. Yet, this issue was missed, because this entry is simply never used and never tested. 

Since it is not a sane situation, we are moving out from there, and making the maintenance workload more focused on what matters for `zstd`.